dmcontent/__init__.py

```diff
@@ -2,4 +2,4 @@
 from .errors import ContentTemplateError, QuestionNotFoundError
 from .questions import ContentQuestion
 
-__version__ = '7.14.1'
+__version__ = '7.15.0'
```

Comment:
You'll need to update the version number as I just merged #97 

```suggestion
__version__ = '7.16.0'
```